### ISSUE TITLE
Email notification that saliva kit has been despatched with instructions on how to use

### ISSUE THREAD

**Issue by ellemindwave** [research-resource/research_resource#22]
As a user, I want to feel informed and ready when it comes to how to complete a saliva sample 

This could be on the Saliva consent page, or it could be in an email which goes out when the user has requested the saliva sample kit to tell them it's on it's way?

**Comment by iteles** [comment]
An email would be preferable as it's something people can refer back to easily. :+1:

A pop-up creates a very awkward user experience on mobile.

**Comment by Cleop** [comment]
We discussed an email being sent when a kit is on its way earlier in the meeting, so perhaps that fits in with that form of notification already.

@ellemindwave do the kits come with their own instructions on how to use them inside the parcel?
Otherwise I'm sure this wouldn't be a problem adding this to the email copy.

I imagine some people might not have the email handy when they decide to use the kit whereas they definitely will have a paper copy of the instructions if it's included. But there's nothing lost by having it in the email.

**Comment by ellemindwave** [comment]
@Cleop Yes, the saliva kits come with their own instructions on how to use them. 

If they already get an email letting them know that their saliva kit is on the way then yes, this seems to fit the need for this notification 👍

**Comment by SimonLab** [comment]
same logic as #20 so once this issue is done this issue should be easier to complete

**Comment by ellemindwave** [comment]
@Cleop @SimonLab - I have checked with the client, and it seems that this email isn't necessary. 

Would it possible to just have a little alert/confirmation when a user enters their address and requests the saliva kit that says 'thank you for requesting your saliva kit, you should receive it in x days' 

I will open a new issue for this.
